### PR TITLE
Improvements to featured-videos

### PR DESCRIPTION
**Pull request by sophieH29**

### CODE REVIEW

File: src/elements/featured-videos.html [hunk 1]
```diff
@@ -26,6 +26,11 @@
 
       .title {
         font-size: 24px;
+        min-width: 400px;
```
Comment (ozasadnyy) [inline src/elements/featured-videos.html]: Have you tried to use margins instead?

File: src/elements/featured-videos.html [hunk 2]
```diff
@@ -166,8 +172,13 @@
...
               <paper-icon-button class="video-play" icon="hoverboard:play" on-tap="playVideo" flex layout horizontal center-center></paper-icon-button>
 
-              <plastic-image id="image[[index]]" class="preview-image" srcset="[[getSessionsImageUrl(block.thumbnail, index)]]" sizing="cover"
-                lazy-load preload fade></plastic-image>
+              <plastic-image
+               id="image[[index]]"
+               class="preview-image"
+               srcset="[[getSessionsImageUrl(block.thumbnail, index)]]"
+               sizing="cover"               
+               style="background-color: {$ webapp.themeColor $};"
```
Comment (ozasadnyy) [inline src/elements/featured-videos.html]: I would suggest to use '--secondary-background-color: #F5F5F5;' We have too big difference between webapp color and background
https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/elements/shared-styles.html#L28

File: src/elements/featured-videos.html (hunk 1 above)
Comment (sophieH29) [inline src/elements/featured-videos.html]: yes, but in this case, I prefer to put width, margin here looks hackier

**Comment by ozasadnyy** [comment]
image

File: src/elements/featured-videos.html (hunk 1 above)
Comment (ozasadnyy) [inline src/elements/featured-videos.html]: With `min-width` you will get horizontal scroll
image

**Comment by ozasadnyy** [comment]
Do you need this header?
image

**Pull request merged**